github/event_types.go

```diff
@@ -653,6 +675,21 @@ type StatusEvent struct {
 	Installation *Installation     `json:"installation,omitempty"`
 }
 
+// TeamEvent is triggered when an organization's team is created, modified or deleted.
+// The Webhook event name is "team".
+//
+// Events of this type are not visible in timelines. These events are only used
+// to trigger hooks.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#teamevent
+type TeamEvent struct {
+	Action  *string       `json:"action,omitempty"`
+	Team    *Team         `json:"team,omitempty"`
+	Changes *TeamChange   `json:"changes,omitempty"`
+	Org     *Organization `json:"organization,omitempty"`
+	Sender  *User         `json:"sender,omitempty"`
```

Comment:
Ah, I didn't know about the implicit fields. That's a great catch.

I didn't separate out the webhook events, because this is an event that is only used for webhooks—it doesn't appear in the timeline.